src/_gettsim/synthetic.py

```diff
@@ -297,6 +297,7 @@ def create_constant_across_households_variables(df, n_adults, n_children, policy
         "m_schul_ausbild": 10.0,
         "m_kind_berücks_zeit": 24.0,
         "m_pfleg_berücks_zeit": 1.0,
+        "elterngeld_zu_verst_eink_vor_geburt_sn": 1000.0,
```

Comment:
Why do we need this but not the net income?

Reply:
Elterngeld was not among the default targets. It is now!